src-built-in/components/toolbar/stores/searchStore.js

```diff
@@ -174,24 +174,26 @@ var Actions = {
 	 * @param {*} e
 	 */
 	handleClose(e) {
-		menuWindow.isShowing(function (err, showing) {
-			if (showing) {
-				console.log("close a window")
-				if (!e && cachedBounds) {
-					finsembleWindow.animate({ transitions: { size: { duration: 150, width: cachedBounds.width } } }, () => {
-						cachedBounds = null;
-					});
+		// Don't call isShowing if menuWindow is not set yet
+		if (menuWindow) {
```

Comment:
Could we not write this as `if (! menuWindow){ return; }` so the commit doesn't touch things unnecessarily?